### PR TITLE
KFSPTS-6344: add cynergy attachment customizations to kfs version

### PR DESCRIPTION
**Pull request by bhutchinson**

### CODE REVIEW

**Comment by bhutchinson** [comment]
retest this please

**Comment by db483** [comment]
Hi Bryan, 

The new unit tests have failed, do you need to update the new test file before I review the code? thank you

**Comment by bhutchinson** [comment]
Daniela,
I'm touching base with ChadB on those failures - they only happen on the Jenkins server, not locally. I may need to update the unit test class, so you could review the other files and that one later, or just wait until later to review everything. It's up to you. I don't expect any changes to the non-test artifacts and only slight, if any changes to the unit test class. Thanks.

**Comment by db483** [comment]
thank you Bryan

**Comment by bhutchinson** [comment]
retest this please

**Comment by bhutchinson** [comment]
@db483 With help from ChadB, I've fixed the unit tests (only a one line change in the test class - line 101), so this should be good for review now. Thanks.

File: src/main/java/org/kuali/kfs/krad/config/KRADConfigurer.java [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    }
+
+    @Override
+    public void addAdditonalToConfig() {
+        configureDataSource();
+    }
+
+    @Override
+    public List<String> getAdditionalSpringFiles() {
+        final String files = ConfigContext.getCurrentContextConfig().getProperty("kfs." + getModuleName() + ".additionalSpringFiles");
```
Comment (db483) [inline src/main/java/org/kuali/kfs/krad/config/KRADConfigurer.java]: This seems like something other institutions would also have to do, can it be included in the kuali co version of the code in the future?

File: src/main/java/org/kuali/kfs/krad/service/impl/AttachmentServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+        String fullPathUniqueFileName = getDocumentDirectory(attachment.getNote().getRemoteObjectIdentifier()) + File.separator + attachment.getAttachmentIdentifier();
+        File attachmentFile = new File(fullPathUniqueFileName);
+        attachmentFile.delete();
+    }
+
+    private String getPendingDirectory() {
+        return this.getDocumentDirectory("");
+    }
+
+    protected String getDocumentDirectory(String objectId) {
```
Comment (db483) [inline src/main/java/org/kuali/kfs/krad/service/impl/AttachmentServiceImpl.java]: The only change in this class is the visibility of thus method, can we create a contribution for that we can remove this in the future?

**Comment by db483** [comment]
Thank you Bryan. Code changes look good and unit tests have passed, I am merging this.

**Pull request merged**